Typed out README.md

G-Thor opened the pull request:
Here's a draft of the README file.
Let me know if you spot any glaring holes.

G-Thor committed:
Typed out README.md

lumpidu commented on README.md:
```diff
@@ -1,2 +1,135 @@
+  - In Flite, these parameters are stored in the `<voicename>_db` variable which is set near the bottom of `flite/<voicename>_clunits.c`
+- Thirdly, the waveform join method may be suboptimal. That is set by the `join_method` parameter in the runtime parameters of Festival. 
+  - Supported values in festival are *windowed*, *none* and *modified_lpc*.
+  - In Flite, only *simple_join*, which corresponds to *none* and *modified_lpc* are implemented, and they are specified in the `register_<voicename>` function in `flite/<voicename>.c`, along with other parameters. It can be set by e.g.:
```
maybe some words about the number of wav. files used for the generation of the voice and some intuition about how the number can be tweaked and what the expected effects in terms of qualitiy vs. voice size means ?

lumpidu commented on README.md:
```diff
@@ -1,2 +1,135 @@
+There are 2 primary voice-specific files in this voice build. They are, relative to the voice directory: 
+- `etc/txt.done.data`, which contains normalized prompts with inserted phrasing tags
+- `festvox/lexicon.scm` which contains a mix of manual and automatic g2p transcriptions of all words that occur in the source data.
+
```
could you add a reference to the paper that stands behind the voice generation ? https://aclanthology.org/2020.sltu-1.44.pdf

lumpidu commented on README.md:
```diff
@@ -1,2 +1,135 @@
+|Y:   |YY            |
+|Yi   |Yi            |
+|Y    |Y             |
+|T    |T             |
```
isn't `pau` an official symbol ? Or is this phrasing already ? But isn't it trained with the current voice as some special phone ?

lumpidu commented on README.md:
```diff
@@ -1,2 +1,135 @@
+- [Edinburgh Speech Tools](https://github.com/festvox/speech_tools)
+- [Festvox](https://github.com/festvox/festvox)
+- [Festival](https://github.com/festvox/festival)
+- And finally, [a modified version of CMU Flite](https://github.com/grammatek/Flite)
```
Could you maybe add somewhere a description of the icelandic voice addition in the modified version of the Flite repository as it's missing there ?

G-Thor committed:
Update README.md

Added a paragraph about the Icelandic Flite integration, another about modifying the number of input samples, a reference to the two Talrómur-related papers, and reiterated silence token usage.
